release-notes/opensearch-release-notes-2.5.0.md

```diff
@@ -0,0 +1,679 @@
+# OpenSearch and OpenSearch Dashboards 2.5.0 Release Notes
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 2.5.0 includes the following feature, enhancement, bug fixes, infrastructure, documentation, maintenance and refactoring updates.
```

Comment:
Fix or fixes?

Reply:
It looks like we have been using `bug fixes` through all previous notes. I would like to keep this as what it was.